tools/bisection/perf-matrix.py

```
@@ -0,0 +1,405 @@
+        for name, key in [(".so", "so"), ("Data", "data"), ("Total", "total")]:
+            ox_kc = _size_val(size_cells, "oxpinyin-kc", key)
+            ox_tk = _size_val(size_cells, "oxpinyin-tkrzw", key)
+            lp_kc = _size_val(size_cells, "libpinyin-kc", key)
+            lp_tk = _size_val(size_cells, "libpinyin-tkrzw", key)
+            ox_r = ox_kc / ox_tk if ox_tk else float("nan")
+            lp_r = lp_kc / lp_tk if lp_tk else float("nan")
+            interaction = ox_r / lp_r if lp_r else float("nan")
+            print(f"| {name} | {interaction:.3f} |")
+
+    print()
+    print("### Decomposition of the 118.1× init gap")
```

Ruff (0.16.2)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)